inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/store/metric/prometheus/StorePrometheusMetric.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.audit.store.metric.prometheus;
+
+import org.apache.inlong.audit.file.ConfigManager;
+import org.apache.inlong.audit.metric.AbstractMetric;
+import org.apache.inlong.audit.store.metric.MetricDimension;
+import org.apache.inlong.audit.store.metric.MetricItem;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.exporter.HTTPServer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.apache.inlong.audit.store.config.ConfigConstants.AUDIT_STORE_SERVER_NAME;
+import static org.apache.inlong.audit.store.config.ConfigConstants.DEFAULT_PROMETHEUS_PORT;
+import static org.apache.inlong.audit.store.config.ConfigConstants.KEY_PROMETHEUS_PORT;
+
+/**
+ * PrometheusMetric
+ */
+public class StorePrometheusMetric extends Collector implements AbstractMetric {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StorePrometheusMetric.class);
+    private static final String HELP_DESCRIPTION = "help";
+
+    private final MetricItem metricItem;
+    private HTTPServer server;
+
+    public StorePrometheusMetric(MetricItem metricItem) {
+        this.metricItem = metricItem;
+        try {
+            server = new HTTPServer(ConfigManager.getInstance().getValue(KEY_PROMETHEUS_PORT, DEFAULT_PROMETHEUS_PORT));
```

Comment:
`server` seems never used

Reply:
After the new version of HTTPServer creates the object, it will start automatically. There is no displayed start operation, so the server seems never used. 
The server has just been optimized to exit gracefully.